dts/bindings/i3c/i3c-device.yaml

```diff
@@ -67,3 +67,8 @@ properties:
     description: |
       Indicates if the device supports the CCC SETAASA. If true, it will
       be used as an optimization for bus initialization.
+
+  v1p0-support:
+    type: boolean
+    description: |
+      Indicates if the device compiles to I3C v1.0.
```

Comment:
why does this need to be a property in DT? isn't it already determinable by compat ?

Reply:
No, it is not. You'd have to read the datasheet of the part to see what spec version it compiles.